index.html

```diff
@@ -341,8 +345,48 @@ <h2>Skills</h2>
           <hr>
         </div>
       </section>
-      <div class="workDivider"></div>
+
       <!-- contact us section -->
+      <section class="contact">
+        <div class="contact-container">
+          <h2>Contact me</h2>
+
+          <p class="contact-P">If you have an application you <br>are interested in developing, a feature that you
+            need to built or a
+            project that needs coding. i'd love to help with it</p>
+
+          <!-- form section -->
+
+          <form action="" method="post" class="contact-form" action="https://formspree.io/f/xknagbbo">
+            <ul class="form-section">
```

Comment:
-  Please ensure that you are able to collect data using the Formspree service. Right now, when I'm trying to do that, I'm receiving the following error :+1:  You can see how it's done from [this tutorial](https://www.youtube.com/watch?v=taa8Kiong0o). 
Ensure that the form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).
action value should be similar to https://formspree.io/f/{form_id}
method value equals POST
![image](https://user-images.githubusercontent.com/95297251/213024941-47898469-ab06-425c-843c-9b5e13f2b0fd.png)

Reply:
okay thank you very much i will see to that right away
@MaganaAsiati